src/wiseagents/agents/coordinator_wise_agents.py

```diff
@@ -101,6 +101,32 @@ def set_response_delivery(self, response_delivery: Callable[[], WiseAgentMessage
         """Set the function to deliver the response to the client."""
         self._response_delivery = response_delivery
 
+class SequentialMemoryCoordinatorWiseAgent(SequentialCoordinatorWiseAgent):
+    yaml_tag = u'!wiseagents.agents.SequentialMemoryCoordinatorWiseAgent'
+    
+    def __init__(self, name: str, description: str, transport: WiseAgentTransport, agents: List[str]):
+        super().__init__(name, description, transport, agents)
+
+    def handle_request(self, request):
+        """
+        Process a request message by passing it to the first agent in the sequence.
+
+        Args:
+            request (WiseAgentMessage): the request message to process
+        """
+        logging.debug(f"Sequential coordinator received request: {request}")
+
+        # Generate a chat ID that will be used to collaborate on this query
+        chat_id = str(uuid.uuid4())
+
+        ctx = WiseAgentRegistry.get_or_create_context(request.context_name)
+        ctx.set_collaboration_type(chat_id, WiseAgentCollaborationType.SEQUENTIAL_MEMORY)
+        ctx.append_chat_completion(chat_uuid=chat_id, messages={"role": "system", "content": self.system_message or self.llm.system_message})
```

Comment:
system_message should be added to the `__init__` method as an `Optional[str]` and it should also be added in the `__new__` method